.github/actions/rollback_release/index.js

```diff
@@ -0,0 +1,43 @@
+const { Octokit } = require("@octokit/core")
+const core = require('@actions/core');
+
+
+async function rollbackRelease() {
+  const octokit = new Octokit({
+      auth: process.env.GITHUB_TOKEN
+    })
+  
+  const { data } = await octokit.request('GET /repos/Bandwidth/api-docs/releases', {
+    owner: 'Bandwidth',
+    repo: 'api-docs'
+  })
+  
+  const cleanedData = []
+  const cleanData = (data) => {
+   for (const x in data) {
+    if (data[x].draft === false && data[x].prerelease === false) {
```

Comment:
We're gathering releases except for `draft` and `prerelease`. What are other possible values? `Released`, `Failed`?

Reply:
These are the only values that I need to exclude, all other releases are published. They're also not values I'm searching through, each release object has a boolean draft and boolean prerelease key so I'm just checking that those are false.